src/presets/classic.js

```diff
@@ -0,0 +1,43 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  theme: {
+    extend: {
+      colors: {
+        border: {
+          light: "#d6d6d6",
+        },
+        canvas: {
+          base: "#ffffff",
+          inverse: "#000000",
+          translucentGrey: "#00000080",
+        },
+        header: "#2164da",
+        headings: "#6d58bb",
+        navItems: "#323232",
+        paragraph: "#344054",
+        prose: "#484848",
+        subtitle: "#344054",
+        subtleLink: "#767676",
+        site: {
+          primary: {
+            DEFAULT: "#6031b6",
+            hover: "#4b268e",
+          },
+          secondary: {
+            DEFAULT: "#4372d6",
+          },
+        },
+      },
```

Comment:
just checking why for classic we put the site colors into the preset but next we dont

Reply:
This is a set of defaults, honestly not too sure about next yet cos we haven't really finalised the set of site-specific colours, but I can add it inside next just in case.

Reply:
Added in 0eef8dc0c1dc9a95fa438b8f72335b322ced07f8.